.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+  - 1.7
```

Comment:
This should be `1.7.x`.  `1.7` just uses 1.7.0: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use